src/pretix/base/models/orders.py

```diff
@@ -2555,6 +2555,27 @@ def resend_link(self, user=None, auth=None):
                 attach_tickets=True
             )
 
+    @property
+    @scopes_disabled()
+    def user_change_allowed(self) -> bool:
+        """
+        Returns whether or not this order can be changed by the attendee.
+        """
+        from .items import ItemAddOn
+
+        if not self.event.settings.change_allow_attendee or not self.order.user_change_allowed:
+            return False
```

Comment:
I am undecided regarding the name. Shouldn’t it be called `attendee_change_allowed`? Currently it does not make a difference as this is only used on the attendee-order-page, but what if – at some point – on the order-overview page we realize, that we need to decide on a per OrderPosition-basis, that an OrderPosition can be changed or not even by the buyer? We would need `user_change_allowed` on an OrderPosition to have context whether it is on an attendee-page or the buyers order-page.

Wouldn’t it make sense to make two `@property` like so:

```suggestion
    @property
    @scopes_disabled()
    def attendee_change_allowed(self) -> bool:
        if not self.event.settings.change_allow_attendee:
            return false
        return self.user_change_allowed

    @property
    @scopes_disabled()
    def user_change_allowed(self) -> bool:
        """
        Returns whether or not this order can be changed by the attendee.
        """
        from .items import ItemAddOn

        if not self.order.user_change_allowed:
            return False
```